### PR TITLE
feat(seamless limits): swap implemented

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR brings swap implementation for cross border limits

## Testing Steps (optional)
You need silver + account and open swap try to swap from NC -> C account limits should be returned and trying to swap over the limit will show error

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -338,6 +339,7 @@ type MessagesType = {
   'components.txlistitem.valueattime': 'Value When {type}'
   'components.txlistitem.watchonly': 'Non-Spendable'
   'copy.account': 'Account'
+  'copy.above_max': '{value} Miximum'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:342]: spelling error

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.stories.tsx [hunk 2]
```diff
@@ -11,6 +11,44 @@ const store = createStore(combineReducers({}))
...
+          },
+          used: {
+            currency: 'GBP',
+            value: '0'
+          }
+        },
+        requiredTier: 2,
+        requirements: ['KYC']
+      }
+    },
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.stories.tsx:51]: why are these values hardcoded? can we get from API?

File: packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/Checkout/index.tsx [hunk 3]
```diff
@@ -415,30 +427,104 @@ const Checkout: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
...
+                  id='copy.swap_minimum_amount'
+                  defaultMessage='To avoid uncesssary fees and network slipage, the minimum amount for this pair is {amount}.'
+                  values={{
+                    amount:
+                      fix === 'FIAT'
+                        ? fiatToString({ unit: walletCurrency, value: fiatMin })
+                        : `${min} ${baseCoinfig.displaySymbol}`
+                  }}
+                />
+              ) : null}
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/Checkout/index.tsx:490]: is a ternary needed?  what about just doing
```
{amtError === 'BELOW_MIN' && (....)
```

could also do below as well if you want but totally up to you as its just a small knitpick

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.stories.tsx (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.stories.tsx:51]: it's storybook ;)

**Review by schnogz — APPROVED** [review]

**Pull request merged**